tests/test_search.py

```diff
@@ -0,0 +1,117 @@
+import os
+
+import unittest
+from unittest.mock import patch
+from flask.testing import FlaskClient
+from iiify.app import app
+from iiify import search
+import json
```

Comment:
Don't think we need this.

Reply:
I've had a look and I think all of these imports are needed.